mongodbatlas/advanced_clusters.go

```diff
@@ -44,27 +44,28 @@ var _ AdvancedClustersService = &AdvancedClustersServiceOp{}
 
 // AdvancedCluster represents MongoDB cluster.
 type AdvancedCluster struct {
-	BackupEnabled                *bool                      `json:"backupEnabled,omitempty"`
-	BiConnector                  *BiConnector               `json:"biConnector,omitempty"`
```

Comment:
Can we avoid formatter changes?

Reply:
I'm not sure that can be done, it's go fmt because the new field name length is larger, I don't think we should override go fmt

Reply:
>  I don't think we should override go fmt

Apologies for lack of clarity. That wasn't my intention. 

I think a good pattern in this case is to add this field as the last element. Since this field is much longer than everything else we can add comment or enter and then rest of the content is not affected:

<img width="1281" alt="Screenshot 2023-10-30 at 15 43 05" src="https://github.com/mongodb/go-client-mongodb-atlas/assets/981838/2934d62a-3df5-44f0-9d51-7d78231b91f7">

Hope that makes sense

Reply:
What I have done:
1. git reset --soft head~2
2. Moved field to the bottom
3. make fmt

Reply:
In this client we like to keep fields alphabetically ordered for easy comparison in case of missing fields